stores/sql/mysql/chain.go

```diff
@@ -210,10 +211,31 @@ func (c chainUpdateTx) FileContractElement(fcid types.FileContractID) (types.V2F
 	return ssql.FileContractElement(c.ctx, c.tx, fcid)
 }
 
+func (c chainUpdateTx) IsKnownContract(fcid types.FileContractID) (bool, error) {
+	if c.known == nil {
```

Comment:
there's only one place where we instantiate an update tx and that's in `ProcessChainUpdate` (obviously twice, once for every db) so I'd create the map there and get rid of this nil check